scripts/verify-correctness.js

```diff
@@ -0,0 +1,81 @@
+const { getOrdersPaginated } = require("../node_modules/@gnosis.pm/dex-contracts/src/onchain_reading")
+const { isOnlySafeOwner } = require("./utils/trading_strategy_helpers")(web3, artifacts)
+const { getMasterCopy } = require("./utils/internals")(web3, artifacts)
+const { toErc20Units } = require("./utils/printing_tools")
+const assert = require("assert")
+
+const argv = require("yargs")
+  .option("brackets", {
+    type: "string",
+    describe:
+      "Trader account addresses for displaying their information, they can be obtained via the script find_bracket_traders",
+    coerce: str => {
+      return str.split(",")
+    },
+  })
+  .option("masterSafe", {
+    type: "string",
+    describe: "The masterSafe in control of the bracket-traders",
+    coerce: str => {
+      return str.split(",")
+    },
```

Comment:
Is this intended to verify brackets for multiple owners at the same time?

Reply:
Yes, it is :)